condDatasetSubmitter.py

```diff
@@ -77,10 +79,18 @@ def createOptionParser():
                      
   (options,args) = parser.parse_args()
 
-#  if not options.gt or not options.run or not options.conds:
-#      parser.error("options --run, --gt and --conds are mandatory")
-  if not options.newgt or not options.gt or not options.run:
-      parser.error("options --newgt, --run, and --gt  are mandatory")
+ #  print "** options args"
+ #  print options
+ #  print args
+ #  print "** options args"
+
+
+  if not options.newgt or not options.gt or not ( options.run or options.runLs):
+      parser.error("options --newgt, --run, [ either: --run  or: --runLs ] and --gt  are mandatory")
+
+  if (options.runLs):
+    options.runLs = eval(options.runLs)
```

Comment:
all evals should be changed to https://docs.python.org/2/library/ast.html#ast.literal_eval for security reasons